app/code/Magento/Eav/Model/TypeLocator/SimpleType.php

```diff
@@ -62,14 +62,4 @@ public function getType($attributeCode, $entityType)
         ];
         return $backendTypeMap[$backendType] ?? TypeProcessor::NORMALIZED_ANY_TYPE;
     }
-
-    /**
-     * Get data Types from service type list
-     *
-     * @return void
-     */
-    public function getAllServiceDataInterfaces()
-    {
-        $this->serviceTypeList->getDataTypes();
-    }
 }
```

Comment:
remove public methods in interfaces may violate rule BC 
https://devdocs.magento.com/contributor-guide/backward-compatible-development/

Reply:
@mrtuvn that's why we are targeting these changes to 2.5-develop, that's the only way to remove deprecated public methods

Reply:
It's safe if i remove all usage of deprecated methods. Ex: setArea i found used in some place

Reply:
removed deprecated mention method